src/property.rs

```diff
@@ -52,12 +51,10 @@ pub trait Property: Send + Sync {
             }
         );
 
-        if description.contains_key("links") {
-            let links = description
-                .get_mut("links")
-                .unwrap()
-                .as_array_mut()
-                .unwrap();
+        if let Some(links) = description
+            .get_mut("links")
+            .map(|links| links.as_array_mut().unwrap())
+        {
```

Comment:
```suggestion
        if let Some(Value::Array(links)) = description.get_mut("links") {
```

What do you think about this? It's not exactly the same semantics, but I think it makes more sense to initialize a non-array value too to the default array than to panic